doc/specifications/io/io_migration_script.md

```diff
@@ -0,0 +1,74 @@
+# IO migration
+
+> Added in 6.7 / 2016.12
```

Comment:
6.8, and you can drop the year syntx, versions are following semantic versioning now _(besides the yearly LTS, aka 2017LTS marketing name)_.